### ISSUE TITLE
Specs fail on elasticsearch 0.19.9

### ISSUE THREAD

**Issue by grantr** [grantr/rubberband#41]
0.19.9 must have changed something to cause this failure, since 0.19.8 passes.

```
Failures:

  1) type and index parameters should search in all indexes
     Failure/Error: @client.count(@username,{:index => "", :type => ""}).should == 6
     ElasticSearch::RequestError:
       (404) {"error":"IndexMissingException[[_all] missing]","status":404}
     # ./spec/../lib/elasticsearch/transport/base_protocol.rb:221:in `handle_error'
     # ./spec/../lib/elasticsearch/transport/base_protocol.rb:82:in `count'
     # ./spec/../lib/elasticsearch/client/abstract_client.rb:56:in `execute'
     # ./spec/../lib/elasticsearch/client/retrying_client.rb:75:in `execute'
     # ./spec/../lib/elasticsearch/client/retrying_client.rb:98:in `clear_execute_retries_count_after'
     # ./spec/../lib/elasticsearch/client/retrying_client.rb:74:in `execute'
     # ./spec/../lib/elasticsearch/client/index.rb:110:in `count'
     # ./spec/type_spec.rb:28:in `(root)'

  2) type and index parameters should search in types grillo,cote of all indexes
     Failure/Error: @client.count(@username,{:index => "",  :type => "grillo,cote"}).should == 4
     ElasticSearch::RequestError:
       (404) {"error":"IndexMissingException[[_all] missing]","status":404}
     # ./spec/../lib/elasticsearch/transport/base_protocol.rb:221:in `handle_error'
     # ./spec/../lib/elasticsearch/transport/base_protocol.rb:82:in `count'
     # ./spec/../lib/elasticsearch/client/abstract_client.rb:56:in `execute'
     # ./spec/../lib/elasticsearch/client/retrying_client.rb:75:in `execute'
     # ./spec/../lib/elasticsearch/client/retrying_client.rb:98:in `clear_execute_retries_count_after'
     # ./spec/../lib/elasticsearch/client/retrying_client.rb:74:in `execute'
     # ./spec/../lib/elasticsearch/client/index.rb:110:in `count'
     # ./spec/type_spec.rb:45:in `(root)'
```

**Comment by grantr** [comment]
Fixed by f58ff6f9a5f572f3290ab166fce7531350d41c6d

Elasticsearch 0.19.9 no longer supports the '_all' syntax to search all indices. '*' is supported though.

**Comment by grantr** [comment]
in v0.9.5
